rfcs/034-location_location_location/README.md

```diff
@@ -0,0 +1,133 @@
+# RFC 034: Modelling Locations in the Catalogue API
+
+As part of upcoming work to add journals and holdings to the Catalogue API, we need to expand the range of locations which are presented by the API.
```

Comment:
Nit: journals are already there, but we don't describe the holdings or how to access online resources.